etc/helm/pachyderm/templates/kube-event-tail/deployment.yaml

```diff
@@ -0,0 +1,69 @@
+{{- if .Values.kubeEventTail.enabled }}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: kube-event-tail
+    suite: pachyderm
+  name: kube-event-tail
+  namespace: {{ .Release.Namespace }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: kube-event-tail
+      suite: pachyderm
+  template:
+    metadata:
+      labels:
+        app: kube-event-tail
+        suite: pachyderm
+    spec:
+      containers:
+        - name: kube-event-tail
+          env:
+            - name: DEBUG_ADDRESS
+              value: 0.0.0.0:8081
+            - name: NAMESPACE
+{{- if eq .Values.kubeEventTail.clusterScope true }}
+              value: ""
+{{ else }}
+              value: {{ .Release.Namespace }}
+{{- end }}
+          image: docker.io/jrockway/kube-event-tail:v0.0.6
+          imagePullPolicy: IfNotPresent
+          livenessProbe:
+            failureThreshold: 3
+            httpGet:
+              path: /healthz
+              port: debug
+              scheme: HTTP
+            periodSeconds: 10
+            successThreshold: 1
+            timeoutSeconds: 1
+          ports:
+            - containerPort: 8081
+              name: debug
+              protocol: TCP
+          readinessProbe:
+            failureThreshold: 3
+            httpGet:
+              path: /healthz
+              port: debug
+              scheme: HTTP
+            periodSeconds: 10
+            successThreshold: 1
+            timeoutSeconds: 1
+          resources:
+            limits:
+              memory: 45Mi
+            requests:
+              cpu: 100m
+              memory: 45Mi
+          terminationMessagePath: /dev/termination-log
+          terminationMessagePolicy: File
+      securityContext:
```

Comment:
tyler has recently merged new default security contexts which you should include here.  these unfortunately have to be configurable (to be turned off).  some customers have a setup where they won't let the pod be submitted if it doesn't match their specs, and some other customers have a setup where they won't let the pod be submitted if it has any security context (they add their own).  it's annoying.